ManatiCR/vitaminau
# VIT-29: Add webform venta productos

# Task [VIT-29](https://manati.atlassian.net/browse/VIT-29?atlOrigin=eyJpIjoiYzYwMGQ1MDcwZGIwNDA5ODkwNzUwOTJlM2EzOGU2MjYiLCJwIjoiaiJ9)
Creation of the "Venta de productos" form.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following url: 
  - http://vitaminau.docker/form/venta-de-productos
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-29](https://manati.atlassian.net/browse/VIT-29?atlOrigin=eyJpIjoiYzYwMGQ1MDcwZGIwNDA5ODkwNzUwOTJlM2EzOGU2MjYiLCJwIjoiaiJ9)
- Fill all the fields of the form and press the save buttom
- Once you press it your information should be saved and the form should redirect you to a thank you page


# Acceptance criteria
- Only authenticated users can access the form.
- The following fields are not required
  - Canales de comercialización -> Directa -> Detalle
  - Canales de comercialización -> Web -> Detalle
